docker-compose-dev.yml

```diff
@@ -25,8 +25,6 @@ services:
     ports:
      - "${WDQS_FRONTEND_PORT}:80"
     restart: ${RESTART}
-    environment:
-      - WIKIBASE_HOST=wikibase.svc # ? see https://github.com/MaRDI4NFDI/portal-compose/issues/35
```

Comment:
If this is an independent issue we should fix that first.

Reply:
this change due to reviving this very old branch, I didn't notice that alvaro had made these changes unrelated to prometheus -> keeping main